.travis.yml

```diff
@@ -30,7 +30,13 @@ script:
     fi
 
 after_success:
-  - sbt ^^$SBT_VERSION orgAfterCISuccess
+  - if [ "$TRAVIS_BRANCH" = "master" -a "$TRAVIS_PULL_REQUEST" = "false" ]; then
+      if grep -q "SNAPSHOT" version.sbt; then
+        sbt ^^$SBT_VERSION publish;
+      else
+        sbt ^^$SBT_VERSION release;
+      fi
+    fi
```

Comment:
I really don't like this and I'm open to ideas.

Reply:
I think it's ok for a temporary solution. For a permanent solution, we could add a new `sbt task` that checks the version internally and executes `publish` or `release`.